content-src/components/DiscoveryStreamComponents/TopSites/_TopSites.scss

```diff
@@ -10,5 +10,77 @@
     .section-top-bar {
       display: none;
     }
+
+    .top-site-outer {
+      padding: 0 12px;
+    }
+
+    .top-sites-list {
+      margin: 0 -12px;
+    }
+  }
+
+  // Only show 6 cards for 2/3 and 1/3
+  .hide-for-narrow {
+    display: none;
+  }
+}
+
+// Only show 8 cards for the full row.
+.ds-column-12 {
+  .ds-top-sites {
+    .hide-for-narrow {
+      display: inline-block;
+    }
+  }
+}
+
+// Size overrides for topsites in the 2/3 view.
+.ds-column-8 {
+  .ds-top-sites {
+
+    .top-site-outer {
+      padding: 0 10px;
+    }
+
+    .top-sites-list {
+      margin: 0 -10px;
+    }
+
+    .top-site-inner {
+      .tile {
+        width: 84.67px;
```

Comment:
These numbers are unusual enough that it probably warrants a comment as to where they come from, in case anyone needs to change it in the future.